includes/Experiments.php

```diff
@@ -274,17 +252,6 @@ public function get_experiments(): array {
 				'description' => __( 'Enable in-progress template actions', 'web-stories' ),
 				'group'       => 'dashboard',
 			],
-			/**
-			 * Author: @brittanyirl
-			 * Issue: 2292
-			 * Creation date: 2020-06-11
-			 */
-			[
-				'name'        => 'enableBookmarkActions',
```

Comment:
Looks like we used that experiment name for a PHP unit test which now fails because it was removed 😅 

https://github.com/google/web-stories-wp/blob/64bc19d15741ec175ca7a8121ce2db8ad1153499/tests/phpunit/tests/Experiments.php#L237-L244

Can you update that test to use a name that still exists?